latest/index.html

```diff
@@ -372,26 +372,27 @@ <h2>Reconciliation Queries</h2>
       <section>
         <h3>Structure of a Reconciliation Query</h3>
         <p>
-          A <dfn>reconciliation query</dfn> consists of:
+          A <dfn>reconciliation query</dfn> consists of the following fields. They are all optional
+	  but at last one of <code>query</code> and <code>properties</code> must be supplied.
           <dl>
 	    <dt><code>query</code></dt>
-            <dd>A query string, consisting of a non-empty string, which is mandatory.
+            <dd>A query string, consisting of a non-empty string.
 	    	By supplying such a string, a client intends to search for entities with similar
 	    	names. The specifics of how this similarity is defined are determined by the service.</dd>
 	    <dt><code>type</code></dt>
-            <dd>Optionally, a list of <a>types</a>. Supplying such types allows users to restrict
+            <dd>A list of <a>types</a>. Supplying such types allows users to restrict
 	    	the search to entities which bear those types. Whether this restriction should be a
 	    	hard constraint or simply induce a change on the reconciliation scores can be
 	    	determined by the service. In particular, services MAY return candidates which
 	    	do not belong to any of the supplied types;</dd>
```

Comment:
Unrelated to this PR, but services may only do this if allowed by `type_strict == "should"`, otherwise they need to consider the `all` or `any` type criteria.